audits/manifest/background-color.js

```diff
@@ -26,13 +26,16 @@ class ManifestBackgroundColor {
     return 'Contains background_color';
   }
 
-  static audit(inputs) {
-    let hasBackgroundColor = false;
-    const manifest = inputs.manifest;
+  static hasBackgroundColorValue(manifest) {
+    /* eslint-disable camelcase */
+    return Boolean(manifest !== undefined &&
```

Comment:
I did it because the moment someone adds a || onto the tail of this expression we are in trouble. But the boolean is more an explicit cast for readability and for future proofing the body.